update-extension-dependencies.sh

```diff
@@ -32,7 +32,7 @@ echo ''
 # get all "artifact-id" values from the generated json file
 # pipefail is switched off briefly so that a better error can be logged when nothing is found
 set +o pipefail
-ARTIFACT_IDS=`grep '"artifact"' devtools/bom-descriptor-json/target/*.json | grep -Po '(?<=io.quarkus:)(?!quarkus-bom)[^:]+' | sort`
+ARTIFACT_IDS=`grep '^    artifact"' devtools/bom-descriptor-json/target/*.json | grep -Po '(?<=io.quarkus:)(?!quarkus-bom)[^:]+' | sort`
```

Comment:
Have you tested this? Because it doesn't work and I'm not sure how it could?

Reply:
Two things AFAICS:
- the removal of `"` before artifact looks suspicious to me?
- by default, grep does not enable regexp. You need `-E` to enable them and I think `^` requires it.

Reply:
I actually tested this locally without `-E` and it worked but the `"` is indeed missing!
See also: https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/New.20catalog.20API/near/230337524

Reply:
yeah my mistake, I just fixed it